Tasks/PowerShellOnTargetMachines/PowerShellJob.ps1

```diff
@@ -9,9 +9,12 @@ param (
     [string]$httpProtocolOption,
     [string]$skipCACheckOption,
     [string]$enableDetailedLogging,
-    [string]$sessionVariables
+    [string]$sessionVariables,
+    [string]$scriptRoot
```

Comment:
Increase path version in both tasks.

Reply:
Done . Increased in task Json file patch version.